hc/api/tests/test_sendalerts.py

```diff
@@ -28,15 +27,9 @@ def test_it_handles_few(self, mock):
             handled_names.append(args[0].name)
 
         assert set(names) == set(handled_names)
-        ### The above assert fails. Make it pass
 
     def test_it_handles_grace_period(self):
         check = Check(user=self.alice, status="up")
-        # 1 day 30 minutes after ping the check is in grace period:
         check.last_ping = timezone.now() - timedelta(days=1, minutes=30)
         check.save()
-
-        # Expect no exceptions--
-        Command().handle_one(check)
-
-    ### Assert when Command's handle many that when handle_many should return True
+        result = Command().handle_one(check)
```

Comment:
This does not look complete.

Reply:
I have removed this test. It was being repeated.